### DISCUSSION TITLE
Disabling cookies breaks the app

### DISCUSSION THREAD

**Discussion by meandillar** [TanStack/query#1175] [General] [answered]
I noticed that disabling cookies crashes my app. I assume because react-query uses local storage.

I'm using next.js with a `ReactQueryCacheProvider` component wrapping my app in `_app.js`

Currently just getting a white screen and `Error was not caught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.` in the console. Would it be an option react-query to handle the error and enable the app to load without any data?

**Comment by tannerlinsley** [comment, marked as answer]
React Query doesn’t use local storage. The devtools does, but AFAIK is guarded against this situation.

**Reply by meandillar** [reply]
Ah yes thanks for the insight. I've done a quick test using the nextjs example from the repo, disabled all my browser extensions and I get the white screen and uncaught error on Chrome and FF. If you think it's worth opening an issue let me know, otherwise it's easy enough to just comment devtools to test local storage functions with cookies disabled.

Cheers for the great library!

**Reply by meandillar** [reply]
I've tested a different example unrelated to nextjs and it works without a problem. The issue looks to be in the next dev environment.
